modules/host/proofs.go

```diff
@@ -0,0 +1,153 @@
+package host
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/NebulousLabs/Sia/consensus"
+	"github.com/NebulousLabs/Sia/hash"
+)
+
+// TODO: Hold off on both storage proofs and deleting files for a few blocks
+// after the first possible opportunity to reduce risk of loss due to
+// blockchain reorganization.
+func (h *Host) threadedConsensusListen(updateChan chan struct{}) {
+	for _ = range updateChan {
```

Comment:
didn't know that

still though would rather wait, since most of our machines are currently on go 1.3